evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -33,14 +37,37 @@
 )
 
 
+GENESIS_COLLATION_HASH = b'\x00' * 32
```

Comment:
Convention has been that this would live in `evm.vm.forks.sharding.constants`

Reply:
Will move it. Thank you so much:)